packages/core/src/awsService/cloudWatchLogs/registry/liveTailSession.ts

```diff
@@ -48,6 +53,9 @@ export class LiveTailSession {
         }
         this._maxLines = LiveTailSession.settings.get('limit', 10000)
         this._uri = createLiveTailURIFromArgs(configuration)
+        this.statusBarItem = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Right, 0)
```

Comment:
👍 this is a secondary/contextual item, so `Right` is the [preferred placement](https://code.visualstudio.com/api/ux-guidelines/status-bar).